.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
+# editorconfig-tools is unable to ignore longs strings or urls
+max_line_length = null
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
MD has significant whitespace, so if you want e.g. a linebreak without starting a new paragraph, you add two trailing spaces.

![image](https://cloud.githubusercontent.com/assets/1404810/20981835/6bfdfee6-bcb6-11e6-8695-45736274b624.png)

Want me to remove it?